### PR TITLE
[Azs.Fabric.Admin] Regenerate.  New Test.  Recorded Sessions.  Bug Fix.

### PR DESCRIPTION
**Pull request by deathly809**
Had an error in the OpenAPI spec.  Fixed that and regenerated the C#.  Added two new tests and re-recorded all session data.  I have updated the version from 0.1.0 to 0.1.1 since this is just a bug fix.

https://github.com/Azure/azure-rest-api-specs/pull/1720

### CODE REVIEW

File: src/AzureStack/FabricAdmin/Fabric.Admin/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -25,5 +25,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("0.1.0.0")]
-[assembly: AssemblyFileVersion("0.1.0.0")]
+[assembly: AssemblyVersion("0.1.1.0")]
```
Comment (shahabhijeet) [inline src/AzureStack/FabricAdmin/Fabric.Admin/Properties/AssemblyInfo.cs]: Please do not update assembly version when you change AssemblyFileVersion.
Keep it as 0.1.0.0

File: src/AzureStack/FabricAdmin/Fabric.Admin/generate.cmd [hunk 2]
```diff
@@ -4,4 +4,4 @@
 ::
 
 @echo off
-call %~dp0..\..\..\..\tools\generate.cmd azsadmin/resource-manager/fabric latest Azure current azure-rest-api-specs %CD%
+call %~dp0..\..\..\..\tools\generate.cmd azsadmin/resource-manager/fabric latest deathly809 current azure-rest-api-specs %CD%
```
Comment (shahabhijeet) [inline src/AzureStack/FabricAdmin/Fabric.Admin/generate.cmd]: do you have the update meta data txt file that get's generated as part of this generated code?
Also we do not allow you to generate code in public repo from your fork.
Generate from main REST API spec fork.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Review by shahabhijeet — DISMISSED** [review]
Please open PR once you have your spec merged and your SDK PR is generated from the main branch of REST SPEC.
We need to review it twice when the SDK is not the final version.

File: src/AzureStack/FabricAdmin/Fabric.Admin/generate.cmd (hunk 2 above)
Comment (deathly809) [reply src/AzureStack/FabricAdmin/Fabric.Admin/generate.cmd]: @shahabhijeet What metadata file?  I have never included any metadata files in my PRs and it is not mentioned anywhere in the readme.md.  If there is something that needs to be done with this file then it should be included in the PR process instructions.  Please update the instructions.

File: src/AzureStack/FabricAdmin/Fabric.Admin/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (deathly809) [reply src/AzureStack/FabricAdmin/Fabric.Admin/Properties/AssemblyInfo.cs]: Done.

**Comment by deathly809** [comment]
@shahabhijeet Fixed the stuff.  I cannot reopen.

**Review by markcowl — APPROVED** [review]

**Pull request merged**